src/workloads/execution/CreateIndex.yml

```diff
@@ -0,0 +1,163 @@
+# This workload inserts documents in the first phase containing all types of indexes. The second phase creates indexes for each of them.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 150000
+    BatchSize: 10000
+    Document:
+      a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
```

Comment:
I think this workload would be easier to understand if the field names were less opaque and representative of their types. e.g. "point", instead of "a".